### PR TITLE
chore: migrates auth category from inquirer to prompter library

### PR DESCRIPTION
**Pull request by lazpavel**
#### Description of changes
- migrates auth category from inquirer to prompter library
#### Checklist
- [x] PR description included
- [ ] `yarn test` passes
- [x] [Pull request labels](https://github.com/aws-amplify/amplify-cli/blob/dev/CONTRIBUTING.md#labels) are added

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by Amplifiyer** [comment]
Are there no unit test changes needed? If there is no existing coverage, is it possible to add for this change?

**Comment by lazpavel** [comment]
@Amplifiyer we already do have e2e tests for auth prompts. I will try to add some unit tests as well. Marking the PR as a draft till I have something

File: packages/amplify-category-auth/src/provider-utils/awscloudformation/index.ts [hunk 1]
```diff
@@ -407,9 +400,9 @@ export const console = async (context: $TSContext, amplifyMeta: any): Promise<an
     } else {
       await openIdentityPoolConsole(context, Region, cognitoOutput.IdentityPoolId);
     }
-    context.print.info('');
+    printer.info('');
```
Comment (edwardfoyle) [inline packages/amplify-category-auth/src/provider-utils/awscloudformation/index.ts:403]: nit: `printer.blankLine()`

File: packages/amplify-category-auth/src/provider-utils/awscloudformation/question-factories/core-questions.ts [hunk 2]
```diff
@@ -49,12 +49,9 @@ export const parseInputs = async (
   const prefix = input.prefix ? `${'\n'} ${questionChalk(input.prefix)} ${'\n'}` : '';
 
   let question: $TSAny = {
```
Comment (edwardfoyle) [inline packages/amplify-category-auth/src/provider-utils/awscloudformation/question-factories/core-questions.ts:51]: don't have full context on where this is used, but it looks like it's still formatted to be an input to inquirer?

Comment (lazpavel) [reply packages/amplify-category-auth/src/provider-utils/awscloudformation/question-factories/core-questions.ts:51]: wrote this helper method to handle the return object from parseInputs: https://github.com/lazpavel/amplify-cli/blob/auth-inquirer-to-prompter/packages/amplify-category-auth/src/provider-utils/awscloudformation/service-walkthroughs/auth-questions.ts#L718-L757

**Pull request closed without merging**